.dotrun.json

```diff
@@ -0,0 +1 @@
+{"yarn": {"dependencies": {}, "installed_packages": {}, "lockfile_hash": "dd07d22cbf9a738bdb68b591bc1bc742"}, "python": {"requirements": "flask==1.1.2\ntalisker[gunicorn,gevent]==0.18.0\n", "installed_packages": ["Flask-1.1.2.dist-info", "pip-21.0.1.dist-info", "contextvars-2.4.dist-info", "statsd-3.3.0.dist-info", "MarkupSafe-1.1.1.dist-info", "Jinja2-2.11.3.dist-info", "Werkzeug-1.0.1.dist-info", "zope.interface-5.3.0.dist-info", "future-0.16.0.dist-info", "immutables-0.15.dist-info", "requests-2.25.1.dist-info", "chardet-4.0.0.dist-info", "idna-2.10.dist-info", "urllib3-1.26.4.dist-info", "zope.event-4.5.0.dist-info", "wheel-0.36.2.dist-info", "talisker-0.18.0.dist-info", "gevent-21.1.2.dist-info", "humanize-3.2.0.dist-info", "certifi-2020.12.5.dist-info", "setuptools-54.0.0.dist-info", "greenlet-1.0.0.dist-info", "itsdangerous-1.1.0.dist-info", "gunicorn-20.1.0.dist-info"]}}
```

Comment:
you might want to add this file in the `.gitignore`

Reply:
reopening the conversation since not solved.
This file should be removed and added in the `.gitignore`

Reply:
Sorry this should be fixed with a rebase, it's already in a previous PR